### ISSUE TITLE
Support line-breaks in MD and HTML formats

### ISSUE THREAD

**Issue by OlegKunitsyn** [brunopacheco1/coboldoc#15]
Input: HTML `<br>` or Markdown `  ` line-breaks.
Output: 
- in MD format COBOLDoc generates 2 spaces
- in MD format COBOLDoc generates `<br>` 

See https://www.markdownguide.org/basic-syntax/#line-breaks

**Comment by brunopacheco1** [comment]
Ah, okay, now I got what you were thinking of. Perhaps supporting in-line MD syntax would be as useful as having in-line HTML. I'll work on that.

**Comment by brunopacheco1** [comment]
I believe it should support in-line Markdown content (that includes double blank spaces in the end of a line).

**Comment by OlegKunitsyn** [comment]
No, i just wanted make html tags optional for braking the line. Two spaces without trailing \n will break it too early.

**Comment by brunopacheco1** [comment]
I found a tiny bug on summary parsing, but the rest should work as expected. Could you check the latest changes?

**Comment by OlegKunitsyn** [comment]
I meant
- https://github.com/brunopacheco1/coboldoc/blob/line-breaks/tests/resources/datetime.cbl
- https://github.com/brunopacheco1/coboldoc/blob/line-breaks/tests/resources/datetime.cbl.expected.md

**Comment by brunopacheco1** [comment]
So, it was almost working with the latest changes, but anyway I found an empty line filtering that could be a problem when exporting to Markdown.

I believe it is working as you are expecting.

**Comment by OlegKunitsyn** [comment]
Thank you! Yes, much better, but "## Function Summary" still has HTML <br> tags.
Coboldoc 1.1.6.

**Comment by brunopacheco1** [comment]
This was on purpose, end-of-line char (`\n`) is breaking Markdown table syntax, so COBOLDoc is replacing deliberately `\s\s\n` by `<br>`.

But do you think this is a problem? Markdown accepts HTML content and this is happening only on the output, it is not visible in the source code.
